armi/reactor/blueprints/reactorBlueprint.py

```diff
@@ -171,34 +170,51 @@ def construct(self, cs, bp, reactor, geom=None, loadAssems=True):
         )
         system.spatialLocator = spatialLocator
         if context.MPI_RANK != 0:
-            # on non-primary nodes we don't bother building up the assemblies
-            # because they will be populated with DistributeState.
+            # Non-primary nodes get the assemblies via DistributeState.
             return None
 
-        # TODO: This is also pretty specific to Core-like things. We envision systems
-        # with non-Core-like structure. Again, probably only doable with subclassing of
-        # Blueprints
+        system = self._constructAssemblies(cs, bp, loadAssems, system, gridDesign)
+
+        # TODO: We should allow for non-Assembly objects/geometries to be loaded into the grid. For
```

Comment:
NOTE: I am _not_ adding a new today. This section of this file had **2** TODOs, I rephrased and reduced it to **1**.